packages/paste-website/src/pages/components/breadcrumb/index.mdx

```diff
@@ -154,6 +154,20 @@ You may choose to use a page Heading as the current page in the breadcrumb.
 </Box>`}
 </LivePreview>
 
+### Breadcrumb with a router
+
+Using breadcrumb with a router is analogous to
+[using Anchor with a router](/components/anchor/#using-anchor-with-a-router), since
+our BreadcrumbItem component creates links using [Anchor](/components/anchor) under the hood.
+
+<iframe
+  src="https://codesandbox.io/embed/using-breadcrumb-with-react-router-t24en?fontsize=14&hidenavigation=1&theme=dark"
+  style="width:100%; height:500px; border:0; border-radius: 4px; overflow:hidden;"
+  title="Using Anchor with React Router"
+  allow="accelerometer; ambient-light-sensor; camera; encrypted-media; geolocation; gyroscope; hid; microphone; midi; payment; usb; vr; xr-spatial-tracking"
+  sandbox="allow-forms allow-modals allow-popups allow-presentation allow-same-origin allow-scripts"
+></iframe>
+
```

Comment:
I don't think there is enough spacing between the sandbox and the next header 

<img width="1113" alt="Screen Shot 2022-01-24 at 11 17 23 AM" src="https://user-images.githubusercontent.com/18120906/150849508-ddbae92a-c416-4e2c-8a90-061fedbfd080.png">

Reply:
V true, just changed it to have a separator between these two. thanks!